survey/src/main/java/com.quickbirdstudios.surveykit/steps/QuestionStep.kt

```diff
@@ -2,13 +2,49 @@ package com.quickbirdstudios.surveykit.steps
+import com.quickbirdstudios.surveykit.AnswerFormat.ScaleAnswerFormat
+import com.quickbirdstudios.surveykit.AnswerFormat.SingleChoiceAnswerFormat
+import com.quickbirdstudios.surveykit.AnswerFormat.TextAnswerFormat
+import com.quickbirdstudios.surveykit.AnswerFormat.TimeAnswerFormat
+import com.quickbirdstudios.surveykit.AnswerFormat.ValuePickerAnswerFormat
 import com.quickbirdstudios.surveykit.StepIdentifier
-import com.quickbirdstudios.surveykit.backend.views.questions.*
+import com.quickbirdstudios.surveykit.backend.views.questions.BooleanQuestionView
+import com.quickbirdstudios.surveykit.backend.views.questions.DatePickerQuestionView
+import com.quickbirdstudios.surveykit.backend.views.questions.DateTimePickerQuestionView
+import com.quickbirdstudios.surveykit.backend.views.questions.DoubleQuestionView
+import com.quickbirdstudios.surveykit.backend.views.questions.EmailQuestionView
+import com.quickbirdstudios.surveykit.backend.views.questions.ImageSelectorQuestionView
+import com.quickbirdstudios.surveykit.backend.views.questions.IntegerQuestionView
+import com.quickbirdstudios.surveykit.backend.views.questions.MultipleChoiceQuestionView
+import com.quickbirdstudios.surveykit.backend.views.questions.ScaleQuestionView
+import com.quickbirdstudios.surveykit.backend.views.questions.SingleChoiceQuestionView
+import com.quickbirdstudios.surveykit.backend.views.questions.TextQuestionView
+import com.quickbirdstudios.surveykit.backend.views.questions.TimePickerQuestionView
+import com.quickbirdstudios.surveykit.backend.views.questions.ValuePickerQuestionView
 import com.quickbirdstudios.surveykit.backend.views.step.QuestionView
 import com.quickbirdstudios.surveykit.result.QuestionResult
 import com.quickbirdstudios.surveykit.result.StepResult
-import com.quickbirdstudios.surveykit.result.question_results.*
+import com.quickbirdstudios.surveykit.result.question_results.BooleanQuestionResult
+import com.quickbirdstudios.surveykit.result.question_results.DateQuestionResult
+import com.quickbirdstudios.surveykit.result.question_results.DateTimeQuestionResult
+import com.quickbirdstudios.surveykit.result.question_results.DoubleQuestionResult
+import com.quickbirdstudios.surveykit.result.question_results.EmailQuestionResult
+import com.quickbirdstudios.surveykit.result.question_results.ImageSelectorResult
+import com.quickbirdstudios.surveykit.result.question_results.IntegerQuestionResult
+import com.quickbirdstudios.surveykit.result.question_results.MultipleChoiceQuestionResult
+import com.quickbirdstudios.surveykit.result.question_results.ScaleQuestionResult
+import com.quickbirdstudios.surveykit.result.question_results.SingleChoiceQuestionResult
+import com.quickbirdstudios.surveykit.result.question_results.TextQuestionResult
+import com.quickbirdstudios.surveykit.result.question_results.TimeQuestionResult
+import com.quickbirdstudios.surveykit.result.question_results.ValuePickerQuestionResult
```

Comment:
Please revert the import expansion. It's not really a big deal, it's just about keeping it the same throughout the project.